Resources/Prototypes/Body/Organs/Animal/animal.yml

```diff
@@ -106,61 +10,3 @@
   - type: Item
```

Comment:
I think ItemComponent is already inherited with these exact values, so this is redundant